src/lib/Microsoft.Health.Common/Auth/Service/ICredentialOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Common.Auth
+{
+    public interface ICredentialOptions
```

Comment:
>ICredentialOptions [](start = 21, length = 18)

Is there a benefit of an empty interface here?  Are you anticipating other types of CredentialOptions in the future?